src/chart/tree/TreeSeries.ts

```diff
@@ -149,7 +149,7 @@ class TreeSeriesModel extends SeriesModel<TreeSeriesOption> {
         function beforeLink(nodeData: List) {
             nodeData.wrapMethod('getItemModel', function (model, idx) {
                 const node = tree.getNodeByDataIndex(idx);
-                if (!node.children.length || !node.isExpand) {
+                if (!node?.children.length || !node.isExpand) {
```

Comment:
We avoid using optional chaining in our code because it is easy to generate lots of unnecessary code after compiled.
```ts
if (!(node && node.children.length && node.isExpand))
``` 
 will be much simpler here.